app/src/main/java/com/mobile/tandil/kotlinbaseproject/MainActivity.kt

```diff
@@ -1,11 +1,38 @@
 package com.mobile.tandil.kotlinbaseproject
 
 import android.os.Bundle
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
+import com.mobile.tandil.kotlinbaseproject.databinding.ActivityMainBinding
+import com.mobile.tandil.kotlinbaseproject.mvvm.viewmodel.MainActivityViewModel
+import com.mobile.tandil.kotlinbaseproject.mvvm.viewmodel.MainActivityViewModel.CheckData
+import com.mobile.tandil.kotlinbaseproject.mvvm.viewmodel.MainActivityViewModel.CheckState
 
 class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+    private val viewModel = MainActivityViewModel()
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        val view = binding.root
```

Comment:
this variable is not needed, please delete it

Reply:
done